### PR TITLE
Update bibdata to rails 6.1

### LINKED ISSUE #1843: Upgrade to rails 6

### PR DESCRIPTION
**Pull request by christinach**
closes #1843

### CODE REVIEW

**Comment by christinach** [comment]
This branch deployed successfully. I'd like to squash the commits and test some indexing before opening it for review.

File: db/schema.rb [hunk 1]
```diff
@@ -2,15 +2,15 @@
...
+# This file is the source Rails uses to define your schema when running `bin/rails
+# db:schema:load`. When creating a new database, `bin/rails db:schema:load` tends to
+# be faster and is potentially less error prone than running all of your
+# migrations from scratch. Old migrations may fail to apply correctly if those
+# migrations use external dependencies or application code.
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2021_09_08_210732) do
+ActiveRecord::Schema.define(version: 2022_05_10_165358) do
```
Comment (christinach) [inline db/schema.rb:13]: @christinach I should reset this since I rolled back the active storage migrations and removed them.

**Comment by christinach** [comment]
Using `deploy@bibdata-alma-worker-staging1:` I tested indexing a single file using the rake task liberate:index_file as:
`SET_URL=http://lib-solr8-staging.princeton.edu:8983/solr/catalog-staging FILE=/home/deploy/second_file.xml bundle exec rake liberate:index_file` and records were indexed.

**Comment by christinach** [comment]
I'm also looking into https://github.com/pulibrary/bibdata/issues/1855 to make sure incremental dumps will be created in bibdata-staging (unrelated so far to rails 6 upgrade) and indexed.

**Comment by christinach** [comment]
https://bibdata-staging.princeton.edu/events receives dump files from the alma sandbox and indexes them as expected.

**Comment by christinach** [comment]
we also indexed a full dump of alma records from the alma sandbox. `11,424,842`

**Comment by christinach** [comment]
I've temporarily pointed https://catalog-staging.princeton.edu/catalog?utf8=%E2%9C%93&search_field=all_fields&q= to the`catalog-staging-rebuild` collection.

**Comment by christinach** [comment]
I also ran successfully the theses rake task, to pull theses from dspace.

**Review by maxkadel — APPROVED** [review]
Looks good

**Review by kevinreiss — APPROVED** [review]

**Pull request merged**